jupyter.py

```diff
@@ -0,0 +1,8 @@
+"""Launch the root jupyter command
+
+Alias to jupyter_core
+"""
+
```

Comment:
I suppose, though I don't expect we will have a reason to make any future releases.